File: ExpenseManager/EM.API/Models/Transaction.cs
using System;

namespace EM.API.Models
{
    public class Transaction
    {
        public Guid TransactionID { get; set; }
        public Guid? FromTransactionID { get; set; }
        public Guid TransactionTypeID { get; set; }
        public string TransactionTypeName { get; set; }
        public Guid FromAccountID { get; set; }
        public string FromAccountName { get; set; }
        public Guid? ToAccountID { get; set; }
        public string ToAccountName { get; set; }
        public Guid? CategoryID { get; set; }
        public string CategoryName { get; set; }
        public Guid? SubCategoryID { get; set; }
        public string SubCategoryName { get; set; }
        public DateTime TransactionDate { get; set; }
        public TimeSpan TransactionTime { get; set; }
        public bool Operator { get; set; }
        public decimal Amount { get; set; }
        public string Note { get; set; }
        public DateTime CreatedDate { get; set; }
    }
}

File: ExpenseManager/EMWEB/src/app/models/transaction.ts
import { Time } from "@angular/common";

export class Transaction {
    TransactionID: string;
    FromTransactionID: string;
    TransactionTypeID: string;
    TransactionTypeName: string;
    FromAccountID: string;
    FromAccountName: string;
    ToAccountID: string;
    ToAccountName: string;
    CategoryID: string;
    CategoryName: string;
    SubCategoryID: string;
    SubCategoryName: string;
    TransactionDate: Date;
    TransactionTime: Time;
    Operator: boolean;
    Amount: number;
    Note: string;
    CreatedDate: Date;
    TransactionIsEdit: boolean;

    constructor(object?:any) {
        if(object) {
            Object.assign(this, object)
        }
    }
}  